module-services/service-desktop/ServiceDesktop.cpp

```diff
@@ -195,6 +200,7 @@ auto ServiceDesktop::usbWorkerInit() -> sys::ReturnCodes
     auto caseColour   = getCaseColour();
 
     LOG_DEBUG("usbWorkerInit Serial Number: %s, Case Colour: %s", serialNumber.c_str(), caseColour.c_str());
+    vTaskDelay(3000);
```

Comment:
How will the system behave when we quickly connect and disconnect the USB plug?

Reply:
This delay shouldn't be there